### ISSUE TITLE
Set canonical link for demo & auth pages

### ISSUE THREAD

**Issue by avivapinchas** [ParabolInc/parabol#6126]
## Issue - Enhancement

Most of our app doesn't get indexed by Google, but we do let Google index the auth and demo pages, since those are 'top of funnel' pages. 

Ahrefs has been flagging these 3 links as 'duplicate content without a canonical link'. 

![Screen Shot 2022-02-24 at 10 29 48 AM](https://user-images.githubusercontent.com/11621016/155566258-a3f0e08b-0471-499a-8d7f-c08ca43712df.png)

We should define https://action.parabol.co/ as the canonical link. 

### Acceptance Criteria (optional)
Ahrefs sees a canonical link for these 3 pages as action.parabol.co 

**Estimated effort:** X points ([see CONTRIBUTING.md](https://github.com/ParabolInc/parabol/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by mattkrick** [comment]
This is a headscratcher.

We have the canonical tag on the site today (also note that the meta content is different, too!) :
![Screen Shot 2022-03-24 at 6 21 19 PM](https://user-images.githubusercontent.com/5514175/160036202-27b377fb-da46-43d1-ad90-20d41fd72a49.png)

seositecheckup.com found the tag:
![Screen Shot 2022-03-24 at 6 20 02 PM](https://user-images.githubusercontent.com/5514175/160036174-f0397cce-7607-4e7a-9c9d-01cd860dc45f.png)

i think ahref may just be wrong, or more accurately, i think they're checking the page as it comes from the server vs. waiting a couple milliseconds before reading the meta tag & the canonical tag. 

As a sanity check, I used google's webcrawler to see what they see, and google sees the canonical tag: https://search.google.com/search-console/inspect?resource_id=sc-domain%3Aaction.parabol.co&id=rTBrLkDNpvHiY6aagq5EpA

tl;dr
google says its there & seositecheckup says its there, so ahref may be wrong about this one

**Comment by accarpenter** [comment]
This is because the page content loads asynchronously through a callback.  Ahrefs scans the page source, not the page after load, which is why it does not see the canonical or the h1.  

If you add the canonical in the same way that you do the title and meta then this issue will be fixed.

**Comment by mattkrick** [comment]
Ooooo, that's a lot of work! As a single page app we send down just a little bit of HTML & then let the javascript do the rest.
doing a server-side render for these pages in particular _is_ possible, but since it already works with google today, we won't see any benefit so I'm not sure we should invest the time since there's not going to be an improvement in SEO.

Not a final decision, if you feel strongly about it let me know!

**Comment by Dschoordsch** [comment]
As a hack we could serve different versions of the index.html based on path without doing any server side rendering just differing in their meta tags.

**Comment by accarpenter** [comment]
It is also probably important to note this is a tooling issue (Ahrefs) and not a dev issue.  As long as Google is registering the correct information any fix would only be for the purposes of making Ahrefs (And other tools that work similarly like Screaming Frog) not give the error - but it probably makes no difference to Google.  You would have to test Bing and other search engines separately if you care about those.

**Comment by mattkrick** [comment]
I think @Dschoordsch has the right idea for how we'd want to do a workaround, although as @accarpenter says, since this already works with google, I'm inclined to say this is working as intended &  I'd suggest we de-prioritize.

I'll remove this from the sprint kanban, but if it turns out that we really wanna optimize for other search engines that don't support async loading we can reopen
